pandas/io/json/_normalize.py

```diff
@@ -531,7 +531,7 @@ def _recursive_extract(data, path, seen_meta, level=0):
             raise ValueError(
                 f"Conflicting metadata name {k}, need distinguishing prefix "
             )
-        result[k] = np.array(v, dtype=object).repeat(lengths)
+        result[k] = np.array(v, dtype=object).repeat(lengths, axis=0).tolist()
```

Comment:
Is the `tolist` necessary for the fix?

Reply:
Yes, otherwise it will raise parsing error.

Reply:
Could you clarify the parsing error? Seem like `tolist` is just a way to force the list assignment to `result[k]` to do type inference?

Reply:
We can't assign 2d `numpy.ndarray` to a `Series`. (Is there any reason for this?) A simple example:

```python
>>>np_arr = np.zeros((2, 3))
>>>df = pd.DataFrame([0, 0], index=[0, 1], columns=["a"])
>>>df["key"] = np_arr
# raise ValueError: Expected a 1D array, got an array with shape (2, 3)
>>>df["key"] = np_arr.tolist()
   a              key
0  0  [0.0, 0.0, 0.0]
1  0  [0.0, 0.0, 0.0]
```

Reply:
I see. I am guessing since a ndarray defines a shape, the shape mismatch raises the `ValueError` while lists get coerced cc @phofl

Reply:
It is correct, that the array assignment does not work, this should check the columns and if they don’t match it raises.

if you want to create a nested list here, I’d rather avoid the array constructor

but yeah, nested data are tricky, I don’t think that there is a straightforward way of assigning a list into every row of a column

Reply:
@GYHHAHA is it possible to avoid the array constructor if we know the values are lists rather than always calling `tolist`?

Reply:
If so, I think we have to implement the same repeat step without numpy. Not sure whether the performance hurt is acceptable (though the function itself is also not effective).

Reply:
IMO if the values are nested, I would say the performance penalty is reasonable since nested values are not natively supported.

Reply:
ok, I will change this.